index.js

```diff
@@ -1,7 +1,7 @@
 function loader (registryOrVersion) {
   const registry = typeof registryOrVersion === 'string' ? require('prismarine-registry')(registryOrVersion) : registryOrVersion
-  const Item = require('prismarine-item')(registry)
```

Comment:
Why was this changed? It is more efficient if prismarine-item can use the same registry version as p window.

Reply:
because of https://github.com/PrismarineJS/prismarine-windows/pull/77

i tested my pr with these changes and my tests failed,
i quickly looked up how to get the mcVersion and implemented that so everything worked again.

i don't know anything about this refactoring i just made my pr compatible with it
if you want you can edit it, but make sure the tests all succeed

Reply:
I will look at it. But I don't see a reason why that would fail. Maybe you where not using up to date dependencies?

Reply:
i think it was because prismarine-item wanted a mcversion string instead of some registry object

Reply:
fixed.